lib/sprockets/vue/script.rb

```diff
@@ -30,7 +39,7 @@ def call(input)
             map = result['sourceMap']
 
             output << "'object' != typeof VComponents && (this.VComponents = {});
-              var module = { exports: null };
+              var module = { exports: null }, exports = {};
```

Comment:
Babel uses the `exports` object to store exports and then assigns `exports['default']` to `module.exports`.